Lesson11_demo.html

```diff
@@ -0,0 +1,262 @@
+      </div>
+      <div class="mb-3">
+        <label for="phone" class="form-label">Phone number</label>
+        <input oninput="checkPhoneNumber()" type="text" class="form-control" id="phone" placeholder="phone number"
+          maxlength="11">
+        <p id="checkphone" class="hide" style="color: red;">Vui long nhap chi co ky tu so, so dau tien la so 0, do dai
+          tu 9
+          den 11!</p>
+      </div>
+      <div class="mb-3">
+        <label for="birthday" class="form-label">Birthday</label>
+        <input type="date" id="birthday" class="form-control" name="birthday">
+      </div>
+      <div class="mb-3" id="checkRadio">
+        <label for="sex" class="form-label">Gioi Tinh</label> <br>
+        <div class="form-check form-check-inline">
+          <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio1" value="Nam"
+            checked="checked">
+          <label class="form-check-label" for="inlineRadio1">Nam</label>
+        </div>
+        <div class="form-check form-check-inline">
+          <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio2" value="Nu">
+          <label class="form-check-label" for="inlineRadio2">Nu</label>
+        </div>
+      </div>
+      <div class="mb-3">
+        <label for="form-label" class="form-label">Address</label>
+        <input class="form-control" type="text" id="text-address" placeholder="Address">
+      </div>
+      <div class="mb-3">
+        <label for="info">Textarea Info</label>
+        <textarea class="form-control" id="info" rows="3"></textarea>
+      </div>
+      <div class="mb-3">
+        <input type="checkbox" name="checkbox" class="form-check-input" id="exampleCheck1">
+        <label class="form-check-label" for="exampleCheck1">Dong y dieu khoan</label>
+      </div>
+      <div class="mb-3">
+        <input type="checkbox" name="checkbox" class="form-check-input" id="exampleCheck">
+        <label class="form-check-label" for="exampleCheck1">Nhan mail tin tuc</label>
+      </div>
+      <div class="mb-3">
+        <button id="submit" onclick="listinf()" type="button" class="btn btn-primary disabled">Register</button>
+        <span  class="form-control" id="text-info" rows="3"></span>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    function checkEmpty() {
+      var birthday = document.getElementById("birthday").value;
+      var address = document.getElementById("text-address").value;
+      var info = document.getElementById("info").value;
+      var exampleCheck1 = document.getElementById("exampleCheck1");
+      // name
+      const regexName = new RegExp(/[^0-9A-Za-z _-]/, 'g');
+      var name = document.getElementById("name").value;
+      var name1 = name.match(regexName);
+      // email
+      const regexEmail = new RegExp(/[^0-9A-Za-z_@.-]/, 'g');
+      var email = document.getElementById("email").value;
```

Comment:
```suggestion
      var emailValue = document.getElementById("email").value;
```
e có thể đặt tên biến như này